### DISCUSSION TITLE
Getting Staging Environment banner with flagsmith

### DISCUSSION THREAD

**Discussion by aftab70** [Flagsmith/flagsmith#2161] [Q&A] [answered]
Hi Team,

I am getting Staging Environment banner after deployment flagsmith with docker composer, How to disable Staging Environment, Please suggest, I did try with given variables.

        - name: DJANGO_SECRET_KEY
          value: "***"

         - name: SECRET_KEY
          value: "***"

**Comment by khvn26** [comment, marked as answer]
Hello!

First of all, I've edited out your secret values for you, but they've likely been already indexed by search engines. Please consider changing them in your configuration.

To disable the banner:

1. Navigate to "Beta Features" section in the left menu.
2. Find the `butter_bar` feature and set an empty value for it.

That should do it.

**Comment by aftab70** [comment]
Thank you its working.
